SPECS/mstflint/0001-update-tool-name-in-error-message.patch

```diff
@@ -0,0 +1,25 @@
+From a61b93f3437347b3f9bbbb38236189eab2f6f73d Mon Sep 17 00:00:00 2001
```

Comment:
Can we actually merge this PR with https://github.com/microsoft/azurelinux/pull/9380/files ? The tool must be the same version as the module for compatibility reasons, especially if the kernel changes are added in first. It also helps with devs for future reference since these are tied with each other.

Reply:
Working in merging both PRs, I will be closing this PR soon